frontend/src/pages/ProblemPage.js

```diff
@@ -387,6 +387,24 @@ function ProblemPage(props) {
         }
     }, [chatHeight]);
 
+    const updateQuestionStatustoAccepted = useCallback(async () => {
+        const data = {userID: '1234', titleSlug: question["titleSlug"]}
+
+        await axios.put(
+            `http://localhost:5000/api/v1/question/history/accepted`, data)
+
+        }, 
+    )
+
+    const updateQuestionStatustoAttempted = useCallback(async () => {
```

Comment:
your callback hook is missing dependency for question.titleslug and userID, also dont use hardcoded userID anymore